src/gui/optionsdialog.cpp

```diff
@@ -488,6 +491,30 @@ void OptionsDialog::initializeLanguageCombo()
     }
 }
 
+void OptionsDialog::initializeStyleCombo()
+{
+		auto changeAvailableStyleList = [&] () {
+			m_ui->stylesCombo->clear();
+			m_ui->stylesCombo->addItems(UiTheme::instance()->availableStyles());	
+		  m_ui->stylesCombo->addItem(tr("Select..."));
+			m_ui->stylesCombo->setCurrentText(UiTheme::instance()->style());
+		};
+		
+		connect(UiTheme::instance(),&UiTheme::availableStylesChanged, changeAvailableStyleList);
+		changeAvailableStyleList();
+
+    connect(m_ui->stylesCombo, &QComboBox::currentTextChanged, [&](const QString &newText) {
+        if (newText == tr("Select...")) {
+            bool success = UiTheme::instance()->installNewStyle(QFileDialog::getOpenFileName(nullptr, 
+																												tr("Select Qss File"),
+                                                        "",
```

Comment:
@jagannatharjun:  In addition to the previously-mentioned indentation issue, this mid-operation blank line (between `nullptr,` and `"",`) seems unnecessary.